### PR TITLE
feat: #609 aip exchange keys 

### PR DESCRIPTION
**Pull request by wjtwzgl12**

### CODE REVIEW

File: app/keys/damoon.pem
Comment (linj121) [inline app/keys/damoon.pem:1]: 😱😱😱 Dangerous

Comment (Wongcheukwai) [reply app/keys/damoon.pem:1]: please be more specific about the comment since the emojis and the word "dangerous" seem like a serious matter.

**Review by Wongcheukwai — COMMENTED** [review]
Hi Ryan, 
Thank you for submitting your PR. Nice work. I have some suggestions: 

1. in the services folder, please capitalise the letter E to align with other places.
2. Could you please add "update function" as I can see this function in  the old version.
3. Could you explain what's aip stands for as I think its seems nebulous to most contributors.

Regards
Kelsey

**Review by Wongcheukwai — CHANGES_REQUESTED** [review]
Hi Ryan,

1. In the services folder, please capitalise the letter E to align with other places.
2. Could you please add "update function" as I can see this function in the old version.
3. Could you explain what aip stands for as I think it seems nebulous to most contributors.
4. When you store you key.Data.access_key and key.Data.secret_key to aipExchangeKeyModel, you have to change them from objects to strings, otherwise it wont work. I've attached the corresponding code from the old version for your reference.

      const keypath = path.join(app.config.baseDir, '/keys/damoon.pub');
      const publicKey = fs.readFileSync(keypath);
      const encryptedBuffer = crypto.publicEncrypt({ key: publicKey }, new Buffer(userMarket.secret));
      const encryptedSecret = encryptedBuffer.toString('base64');
      userMarket.secret_show = userMarket.secret.substr(0, 4) + '******' + userMarket.secret.substr(-4);
      userMarket.secret = encryptedSecret;
      
Regards
Kelsey

File: app/keys/damoon.pem
Comment (linj121) [reply app/keys/damoon.pem:1]: Uploading private key to a public repository can lead to security issues

**Pull request closed without merging**